doc/api_package.rst

```diff
@@ -31,6 +31,10 @@
 
     Seconds since the epoch when the package was built (integer).
 
+  .. attribute:: chksum
+
+    Tuple with package checksum and checksum type or None.
+
```

Comment:
Can the description be updated to include information that this only works when working with package from repository, not with installed package, that it is the checksum of the `.rpm` file in that repository, and that this value is also the `@pkgid` value which links primary metadata with other repository metadata records?